modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,106 @@
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-704]
+== Release 7.0.4 (May 2022)
+Couchbase Server 7.0.4 was released in May 2022.
+This maintenance release contains fixes to issues.
+
+[#fixed-issues-704]
```

Comment:
Do we also have a section for known issues (e.g. https://issues.couchbase.com/browse/MB-51699)?

Reply:
There is a section for Known Issues at the end of the 7.0.4 section. It contains only MB-52313. Do you wish to add MB-51699?

Reply:
Yes, please add it. We've also provided a summary and the workaround on the ticket.